ledger/src/blockstore_db.rs

```diff
@@ -771,42 +814,122 @@ impl ProtobufColumn for columns::TransactionStatus {
     type Type = generated::TransactionStatusMeta;
 }
 
-impl Column for columns::AddressSignatures {
-    type Index = (u64, Pubkey, Slot, Signature);
+impl ColumnIndexDeprecation for columns::TransactionStatus {
+    const DEPRECATED_INDEX_LEN: usize = 80;
+    const CURRENT_INDEX_LEN: usize = 72;
+    type DeprecatedIndex = (u64, Signature, Slot);
```

Comment:
I like it, because I can make the default implementations on the trait. I had considered this, but mistakenly thought I'd have to define the serialization format.

Reply:
Done, squashed into commit adding trait, and rebased trait implementation commits